src/server.ts

```diff
@@ -15,7 +16,10 @@ const start = async () => {
     secret: process.env.PAYLOAD_SECRET,
     express: app,
     onInit: async () => {
-      payload.logger.info(`Payload Admin URL: ${payload.getAdminURL()}`)
+      payload.logger.info(`Payload Admin URL: http://localhost:3000${payload.getAdminURL()}`)
```

Comment:
The `localhost:3000` won't be true on production deployments, but it's nice to have a real URL here...

Reply:
I don't know how to do this. Maybe this is ok for now?